### PR TITLE
 Fixed Mob Duplicator and Animal Sewer not waste energy 

### PR DESCRIPTION
**Pull request by jansamohyl**
Hi,

I fixed the miscalculation in Mob Duplicator which caused it to run (and waste energy) when no mobs could have been spawned, and also made Animal Sewer not waste energy if there are no animals.

Best,
Jan Samohyl

### CODE REVIEW

File: src/main/java/com/buuz135/industrial/tile/mob/MobDuplicatorTile.java [hunk 1]
```diff
@@ -198,7 +199,7 @@ public NBTTagCompound writeToNBT(@NotNull NBTTagCompound compound) {
 
     private boolean canEntitySpawn(EntityLiving living) {
         return /*getWorkingArea().contains(new Vec3d(living.getEntityBoundingBox().minX, living.getEntityBoundingBox().minY, living.getEntityBoundingBox().minZ)) &&
-                getWorkingArea().contains(new Vec3d(living.getEntityBoundingBox().maxX, living.getEntityBoundingBox().maxY, living.getEntityBoundingBox().maxZ)
+                getWorkingArea().contains(new Vec3d(living.getEntityBoundingBox().maxX, living.getEntityBoundingBox().maxY, living.getEntityBoundingBox().maxZ))
```
[the remark this answers is not in the document]
Comment (jansamohyl) [reply src/main/java/com/buuz135/industrial/tile/mob/MobDuplicatorTile.java]: Hi, don't worry about that, it's just fixing syntax in commented code. I think Eclipse fixed it by accident.

File: src/main/java/com/buuz135/industrial/utils/ItemStackUtils.java [hunk 2]
```diff
@@ -123,7 +123,7 @@ public static void renderEntity(int posX, int posY, int scale, float mouseX, flo
         RenderHelper.enableStandardItemLighting();
         //GlStateManager.rotate(-135.0F, 0.0F, 1.0F, 0.0F);
         // GlStateManager.rotate(-((float)Math.atan((double)(mouseY / 40.0F))) * 20.0F, 1.0F, 0.0F, 0.0F);
-//       ent.renderYawOffset = (float)Math.atan((double)(mouseX / 40.0F)) * 20.0F;
+        //       ent.renderYawOffset = (float)Math.atan((double)(mouseX / 40.0F)) * 20.0F;
```
[the remark this answers is not in the document]
Comment (jansamohyl) [reply src/main/java/com/buuz135/industrial/utils/ItemStackUtils.java]: Again, it's just formatting that slipped in (I was trying to get Eclipse to format the code properly).

File: src/main/java/com/buuz135/industrial/tile/mob/MobDuplicatorTile.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheCodedOne) [reply src/main/java/com/buuz135/industrial/tile/mob/MobDuplicatorTile.java]: Well can you remove it, its irrelevant to the PR

[the remark this answers is not in the document]
Comment (jansamohyl) [reply src/main/java/com/buuz135/industrial/tile/mob/MobDuplicatorTile.java]: It is, but I don't want to bother with it. I don't see why I would intentionally leave a syntax error there.

**Comment by jansamohyl** [comment]
I guess it's take it or leave it, as I already maintain my own version of IF anyway. I don't think it's an enhancement, but I don't mind it being marked as such. I really don't like to bikeshed about things.

In any case, I appreciate the mod and the work you're doing.

**Pull request merged**